ui_framework/components/modal/confirm_modal.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import { requiredProps } from '../../test/required_props';
+
+import {
+  KuiConfirmModal,
+} from './confirm_modal';
+
+const onConfirm = sinon.spy();
+const onCancel = sinon.spy();
+
+test('renders KuiConfirmModal', () => {
+  const component = mount(<KuiConfirmModal
```

Comment:
What does the snapshot look like if you shallow render this instead? (As it's built up of purely lower-level components it might make it include "less noise")